polling_stations/apps/councils/models.py

```diff
@@ -25,19 +25,37 @@ def with_polling_stations_in_db(self):
             .exclude(pollingstation=None)
         )
 
+    def with_future_upload_details(self):
+        all_valid_subquery = (
+            File.objects.filter(upload_id=OuterRef("pk"))
+            .annotate(all_valid=Func("csv_valid", function="BOOL_AND"))
+            .values("all_valid")
+        )
+        upload_subquery = (
+            Upload.objects.future()
+            .filter(gss=OuterRef("council_id"))
+            .annotate(all_files_valid=Subquery(all_valid_subquery))
+            .order_by("-timestamp")
+        )
+        return self.using(DEFAULT_DB_ALIAS).annotate(
```

Comment:
Can you explain why you're using `using` here?

Reply:
I mostly did it because it's what was there on the similar `with_ems_from_uploads` method, and then got distracted by everything else and never questioned it!

Without it I think it will hit RDS because the file_upload models are routed there. If that's the case it will break in a confusing way if replication isn't working, so maybe better to just let it go to RDS, though I think that will slow page loads a little. 

However, I don't actually know for sure how the routing works, and it could be the case that it would go to the local db anyway, since `councils.Council` isn't special cased in  `db_routers.py`. Do you know?